templates/rbac.yaml

```diff
@@ -14,48 +14,6 @@ metadata:
     {{ $key }}: {{ $value | quote }}
   {{- end }}
 ---
-# The roles the cemanager needs cluster wide for
-# checking available resources.
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: coder-{{ .Release.Namespace }}-cluster
-rules:
-  - apiGroups:
-      - "" # indicates the core API group
-    resources:
-      # Lets us determine the cores/RAM available on a node
-      # to avoid letting a user select an unschedulable number.
-      - nodes
-      - pods
-      # Lets us determine what namespaces are available for
-      # scheduling. The permissions for a namespace are handled
-      # via regular namespaced Roles.
-      - namespaces
-    # We request read-only permissions at the cluster level to prevent
-    # the product from making any surprising mutations.
-    verbs:
-      - get
-      - list
----
-# Bind the cluster role to the coder serviceaccount.
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  namespace: {{ .Release.Namespace | quote }}
-  name: coder-{{ .Release.Namespace }}-clusterbinding
-subjects:
-  - kind: ServiceAccount
-    name: coder
-    namespace: {{ .Release.Namespace | quote }}
-roleRef:
-  kind: ClusterRole
-  name: coder-{{ .Release.Namespace }}-cluster
-  apiGroup: rbac.authorization.k8s.io
-
-{{- $namespaces := append .Values.namespaceWhitelist .Release.Namespace }}
-{{- range $namespaces }}
----
```

Comment:
You can't delete this yet unless you delete the deprecated `namespaceWhitelist` value. Deleting that value has the consequence that any old workspaces that haven't been migrated to workspace providers become inaccessible by coder. We probably need to add a big scary banner to the workspaces page for a few releases before doing that